### PR TITLE
Apply contribution guidelines for "positive" rule

### PR DESCRIPTION
**Pull request by IsmaelElias**

### CODE REVIEW

File: library/Exceptions/PositiveException.php [hunk 1]
```diff
@@ -13,8 +13,17 @@
 
 namespace Respect\Validation\Exceptions;
 
-class PositiveException extends ValidationException
+/**
+ * @author Alexandre <alexandre@gaigalas.net>
```
Comment (williamespindola) [inline library/Exceptions/PositiveException.php]: you can keep only Alexandre Gomes Gaigalas

File: library/Rules/Positive.php [hunk 2]
```diff
@@ -13,8 +13,19 @@
 
 namespace Respect\Validation\Rules;
 
-class Positive extends AbstractRule
+/**
+ * Validates whether the characters in the input are positive.
+ *
+ * @author Alexandre <alexandre@gaigalas.net>
```
Comment (williamespindola) [inline library/Rules/Positive.php]: 🔪 Alexandre, keep only Alexandre Gomes Gaigalas

File: tests/unit/Rules/PositiveTest.php [hunk 3]
```diff
@@ -13,65 +13,54 @@
...
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Rules\Positive
  * @covers \Respect\Validation\Exceptions\PositiveException
```
Comment (williamespindola) [inline tests/unit/Rules/PositiveTest.php]: Please remove this `@covers \Respect\Validation\Exceptions\PositiveException` exceptions are tested on integrations test

File: tests/unit/Rules/PositiveTest.php [hunk 4]
```diff
@@ -13,65 +13,54 @@
 
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
```
Comment (williamespindola) [inline tests/unit/Rules/PositiveTest.php]: Please remove this extra space

File: library/Rules/Positive.php [hunk 5]
```diff
@@ -13,8 +13,18 @@
 
 namespace Respect\Validation\Rules;
 
-class Positive extends AbstractRule
+/**
+ * Validates whether the characters in the input are positive.
```
Comment (henriquemoody) [inline library/Rules/Positive.php]:
```diff
-- Validates whether the characters in the input are positive.
++ Validates whether the input is a positive number.
```

File: tests/integration/rules/positive.phpt [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    v::not(v::positive())->assert('165');
+} catch (NestedValidationException $exception) {
+    echo $exception->getFullMessage().PHP_EOL;
+}
+
+?>
+--EXPECTF--
+-10 must be positive
+- "a" must be positive
+16 must not be positive
```
Comment (henriquemoody) [inline tests/integration/rules/positive.phpt]: I prefer to first test `check()`, and then `assert()`, it makes easy to read.

**Comment by henriquemoody** [comment]
You can find your commit in master (72933a718f6c0ca073e3221cb4a9403243809462)!

**Pull request closed without merging**